### ISSUE TITLE
How can I explore without searching?

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/openverse#2710]
## Problem

The homepage says I can explore the 700 million websites, but I can't seem to do it without a search term. Could we lead with some tags/categories somewhere that pulls me into the "explore" flow, similar to the empty result set view?

**Homepage**
<img width="400" alt="Screenshot 2023-07-25 at 10 39 36 AM" src="https://github.com/WordPress/openverse/assets/1657336/3405fdbf-eeba-44ed-85ff-add6deb6db2b">

**Empty result set**
| Empty |
|---|
| <img width="721" alt="Screenshot 2023-07-25 at 10 45 38 AM" src="https://github.com/WordPress/openverse/assets/1657336/6e9bc58d-0ffd-426d-937b-f069818fafa9"> |

## Related
Since openverse is Search Term centric, when I search from something that has no results, there is no way to get out of that flow until i think of a new search term, or use the provided search term hints in the content.
| GIF: Desktop |
| --- |
| ![Screen Capture on 2023-07-25 at 10-45-30](https://github.com/WordPress/openverse/assets/1657336/0eb7024a-9da9-4d7f-b1b7-5df933e0f5df) |

| GIF: Mobile |
| --- |
| ![Screen Capture on 2023-07-25 at 10-44-43](https://github.com/WordPress/openverse/assets/1657336/68587f52-5d2c-4a6f-b4ba-0ca7495f3e8d) |

**Comment by AetherUnbound** [comment]
Thanks for sharing your perspective as a user @StevenDufresne! We're actually working on a project currently which we hope will help with exactly this particular use case, you can see our progress & discussions on it here: https://github.com/WordPress/openverse/issues/410. This effort will add several new ways to explore results, including by **providers**, **tags**, and **creators**, without having to use a specific search term. If you have any input you'd like to share on those designs or related issues, it'd be more than welcome!

**Comment by dhruvkb** [comment]
We have implemented additional search views, and one can enter the exploration flow by clicking any source name on the [sources page](https://openverse.org/sources). We've also discussed refreshing the homepage gallery in #5293 which will provide several different entrypoints for exploration.

With those, I think we can close this issue. Thanks again for the suggestion @StevenDufresne and we look forward to your feedback on the approaches we're taking.
